pylinac/winston_lutz.py

```diff
@@ -179,13 +196,20 @@ def max_distance_to_lines(p, lines) -> float:
         result = optimize.minimize(max_distance_to_lines, initial_guess, args=things, bounds=bounds)
         return result
 
+    @property
+    def json(self):
```

Comment:
This isn't JSON, it's a dictionary. Of the two options, I prefer the `results_dict` because it specifies it contains results, but I think we could generalize it one step further with `results_data`. This gives the option of returning one or more types with parameters (e.g. `results_data(as_dict=True, as_json=False)` returns a dict with options of returning something else.

Reply:
+1 on `results_data` or similar instead of `json`.

Reply:
I like that, `results_data` it is. I was always looking for a better name